packages/scaffold-ui/src/partials/w3m-activity-list/index.ts

```diff
@@ -66,6 +67,12 @@ export class W3mActivityList extends LitElement {
   }
 
   public override firstUpdated() {
+    if (ChainController.state.activeChain === 'solana') {
```

Comment:
We should use `ConstantUtils.chain.EVM` from `@web3modal/common` to not use hard coded strings